app/models/crime_application.rb

```diff
@@ -33,6 +33,12 @@ def means_passported?
     !means_passport.empty?
   end
 
+  def passporting_benefit?
+    means_passport.include?('on_benefit_check')
+  end
+
+  delegate :benefit_type, to: :applicant
```

Comment:
There is already a `delegate :date_of_birth`, maybe worth adding `benefit_type` to that same delegate. It uses prefix tho, so it will become `applicant_benefit_type`, unsure if that causes any trouble, seems fine to me. Just a suggestion 👍

Another suggestion if possible is to use the schemas type instead of a string here `means_passport.include?('on_benefit_check')`